run/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM node:10.15-slim
```

Comment:
Sounds like we're moving `Dockerfile` to `run/Dockerfile`? Any specific reason for subdir?

Reply:
Just to differentiate it from the one in `dev`. Perhaps a better name for this one is `build`? I'm not really sure what a good name is.